test/assets.test.ts

```diff
@@ -7,6 +7,7 @@ import { should } from 'chai';
 import * as path from 'path';
 import * as protocol from '../src/omnisharp/protocol';
 import { AssetGenerator } from '../src/assets';
+import * as json from 'comment-json';
```

Comment:
>json [](start = 12, length = 4)

We use 'tolerantParse' inside the extension. We might want to use that here also as I believe this is what VS Code uses.